src/jansson.h

```diff
@@ -186,11 +187,12 @@ static JSON_INLINE enum json_error_code json_error_code(const json_error_t *e) {
 
 void json_object_seed(size_t seed);
 size_t json_object_size(const json_t *object);
-json_t *json_object_get(const json_t *object, const char *key)
+json_t *json_object_getn(const json_t *object, const char *key, size_t key_len)
     JANSSON_ATTRS((warn_unused_result));
-int json_object_set_new(json_t *object, const char *key, json_t *value);
-int json_object_set_new_nocheck(json_t *object, const char *key, json_t *value);
-int json_object_del(json_t *object, const char *key);
+int json_object_setn_new(json_t *object, const char *key, size_t key_len, json_t *value);
+int json_object_setn_new_nocheck(json_t *object, const char *key, size_t key_len,
+                                 json_t *value);
+int json_object_deln(json_t *object, const char *key, size_t key_len);
```

Comment:
I'm afraid that it's not a good idea to remove the export api function

Reply:
also thought about it, returned it back.